frontend/next.config.js

```diff
@@ -2,7 +2,7 @@
   THIS IS BURNED AT BUILDTIME - ENV ARE IGNORED AT RUNTIME
 */
 
```

Comment:
I don't know if it's important (and I deleted some of it by being a touch over-eager), but all the next config examples look like this:
```javascript
/**
 * @type {import('next').NextConfig}
 */
const nextConfig = {
  /* config options here */
}

module.exports = nextConfig
```

Reply:
I think you are right - let me put this in :+1: